src/NTH.Windows.Forms/NTH.Windows.Forms/NativeMethods.cs

```diff
@@ -20,12 +21,21 @@ internal static class NativeMethods
         [DllImport(User32, CharSet = CharSet.Unicode)]
         internal static extern IntPtr SendMessage(IntPtr windowHandle, WindowMessage message, IntPtr wParam, [MarshalAs(UnmanagedType.LPWStr)] string lParam);
 
+        [DllImport(User32, CharSet = CharSet.Auto)]
+        public static extern IntPtr SendMessage(IntPtr hWnd, int msg, IntPtr wParam, IntPtr lParam);
+
+        [DllImport("user32.dll", CharSet = CharSet.Auto)]
```

Comment:
Oops, sorry, I actually just really cared about the constants in the previous declarations but there I forgot it, sorry.